.github/scripts/process-logs-zombienet-sdk.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```

Comment:
Would be nice to have a description what the script does.

Reply:
Done